kolibri/plugins/learn/assets/src/my_downloads/views/DownloadsList/index.vue

```diff
@@ -239,18 +237,28 @@
         },
       },
       paginatedDownloads() {
-        const startIndex = (this.currentPage - 1) * this.itemsPerPage;
-        const endIndex = startIndex + this.itemsPerPage;
-        return this.downloads.slice(startIndex, endIndex);
+        if (this.downloads && this.downloads.length > 0) {
+          const startIndex = (this.currentPage - 1) * this.itemsPerPage;
+          const endIndex = startIndex + this.itemsPerPage;
+          return this.downloads.slice(startIndex, endIndex);
+        } else {
+          return [];
+        }
+      },
+      downloads() {
+        return this.sortedFilteredDownloads;
```

Comment:
This is a function, not a computed prop.

Also, if you want to do this mapping, you can just map in the object that is the return value from `setup`:

```
downloads: sortedFilteredDownloads,
```